src/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -119,5 +133,18 @@ export const AuctionDetailsFragmentContainer = createFragmentContainer(
         ...RegisterButton_me
       }
     `,
+    viewer: graphql`
+      fragment AuctionDetails_viewer on Viewer
+        @argumentDefinitions(saleID: { type: "String" }) {
+        auctionDetailsSaleArtworks: saleArtworksConnection(
+          saleSlug: $saleID
+          aggregations: [TOTAL]
+        ) {
+          counts {
+            total
+          }
+        }
+      }
+    `,
```

Comment:
`totalCount` for the query below always returns `null`. For this reason, data is requested from the `viewer`

```graphql
{
	sale(id: "saleID") {
		artworksConnection(first: 10) {
			totalCount
		}
	}
}
```